src/utils/openmc/material.py

```diff
@@ -76,7 +76,9 @@ def set_id(self, material_id=None):
 
         # If the Material already has an ID, remove it from global list
         if not self._id is None:
-            MATERIAL_IDS.remove(self._id)
+            try:
```

Comment:
Cool - if you don't need this change that that's would be preferable.

One other question for you - have you tested this out with the latest
version of OpenCG? Just make sure to update your fork before building
BEAVRS.
On Mar 25, 2015 9:07 AM, "Nicholas Horelik" <notifications@github.com>
wrote:

> In src/utils/openmc/material.py
> <https://github.com/wbinventor/openmc/pull/7#discussion_r27135821>:
>
> > @@ -76,7 +76,9 @@ def set_id(self, material_id=None):
> >
> >          # If the Material already has an ID, remove it from global list
> >          if not self._id is None:
> > -            MATERIAL_IDS.remove(self._id)
> > +            try:
>
> If i remember correctly that is what I had been doing at one point, but
> now that I look through the cleaned-up BEAVRS scripts again it seems we no
> longer are doing that (and shouldn't be - set_id should always be used, I
> beleive). I'm thinking I'll actually remove this change.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wbinventor/openmc/pull/7/files#r27135821>.
>